Common/include/config_structure.hpp

```diff
@@ -2079,12 +2102,68 @@ class CConfig {
 	 * \return Value of the angle of attack.
 	 */
 	su2double GetAoA(void);
+  
+  /*!
+   * \brief Get the angle of attack of the body. This is the angle between a reference line on a lifting body
+   *        (often the chord line of an airfoil) and the vector representing the relative motion between the
+   *        lifting body and the fluid through which it is moving.
+   * \return Value of the angle of attack.
+   */
+  su2double GetiH(void);
+  
+  /*!
+   * \brief Get the angle of attack of the body. This is the angle between a reference line on a lifting body
```

Comment:
I know it's a pain in the rear, but can you please update your doxy comments for these new functions? There are a bunch, and they aren't all immediately obvious to me by name...